.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: script
+
+env:
+  global:
+     CF_TARGET_URL="https://api.ng.bluemix.net"
+
+services:
```

Comment:
should this services be empty?  if so, is it required?

Reply:
removed